dbms/src/Storages/Page/V3/tests/gtest_blob_store.cpp

```diff
@@ -67,13 +67,14 @@ try
 {
     BlobStats stats(logger, delegator, config);
 
-    BlobFileId file_id1 = 10;
-    BlobFileId file_id2 = 12;
+    BlobFileId file_id1 = 1;
+    BlobFileId file_id2 = 2;
 
-    const auto & path = getTemporaryPath();
-    createIfNotExist(path);
-    Poco::File(fmt::format("{}/{}{}", path, BlobFile::BLOB_PREFIX_NAME, file_id1)).createFile();
-    Poco::File(fmt::format("{}/{}{}", path, BlobFile::BLOB_PREFIX_NAME, file_id2)).createFile();
+    {
+        const auto & lock = stats.lock();
+        stats.createStat(file_id1, lock);
+        stats.createStat(file_id2, lock);
```

Comment:
maybe we should use `createStatNotChecking` here, and don't change the `file_id1` and `file_id2`?